src/directory/mmap_directory.rs

```diff
@@ -610,6 +610,7 @@ mod tests {
             assert!(num_segments <= 4);
             let num_components_except_deletes_and_tempstore =
                 crate::core::SegmentComponent::iterator().len() - 2;
+            std::thread::sleep(Duration::from_millis(100));
```

Comment:
what is this?

Reply:
the mmap_released test is still flaky, that was a test to fix it.